DOCS.md

```diff
@@ -69,270 +70,325 @@ In a Sponsored Ad Break flow, once the user hits a mid-roll break with a true[X]
 
 ![choice card](media/choice_card.png)
 
-***Fig. C** example true\[X\] preroll choice card (full-stream replacement)*
+_**Fig. D** example true[X] pre-roll choice card (full-stream replacement)_
 
-Similarly, once if the user opts-in and completes the true[X] ad, they will be skipped over the remainder of the pre-roll ad break. However, every subsequent mid-roll break in the current stream will also be skipped over. In this case instead of the regular pod of video ads, the user will be shown a "hero card" (also known as a "skip card"):
+Similarly, if the user opts-in and completes the true[X] ad, they will be skipped over the remainder of the pre-roll ad break. However, every subsequent mid-roll break in the current stream will also be skipped over. In this case instead of the regular pod of video ads, the user will be shown a "hero card" (also known as a "skip card"):
 
 ![skip card](media/skip_card.png)
 
-***Fig. D** example true\[X\] mid-roll skip card*
+_**Fig. E** example true[X] mid-roll skip card_
 
 This messaging will be displayed to the user for several seconds, after which they will be returned directly to content.
 
 
 ## How to use TruexAdRenderer
 
-When referring to data returned via Uplynk Preplay, I will reference this example JSON response in order to clearly explain which piece of data is required:
+### When to show true[X]
 
-```
-    https://gist.github.com/jalbini/7c0f908064ae1468beb0e4a11fd187ec
-```
+Upon receiving an ad schedule from your SSAI service, you should be able to detect whether or not true[X] is returned in any of the pods. true[X] ads should have `apiFramework` set to `VPAID` or `truex`.
 
-I will refer to entire object as `response`, and will point to specific pieces of data using standard JSON notation.
+SSAI vendors differ in the way they convey information back about ad schedules to clients. Certain vendors such as Verizon / Uplynk expose API’s which return details about the ad schedule in a JSON object. For other vendors, for instance Google DAI, the true[X] payload will be encapsulated as part of a companion payload on the returned VAST ad. Please work with your true[X] point of contact if you have difficulty identifying the right approach to detecting the true[X] placeholder, which will be the trigger point for the ad experience.
 
+Once the player reaches a true[X] placeholder, it should pause, instantiate the `TruexAdRenderer` and immediately call `init` followed by `start`.
 
-### When to show true\[X\]
+Alternatively, you can instantiate and `init` the `TruexAdRenderer` in preparation for an upcoming placeholder. This will give the `TruexAdRenderer` more time to complete its initial ad request, and will help streamline true[X] load time and minimize wait time for your users. Once the player reaches a placeholder, it can then call `start` to notify the renderer that it can display the unit to the user.
```

Comment:
Hmmm, I suppose this addresses the above comment ☝️ , but I may prefer we commend waiting for the `delegate`. What are your thoughts?